Update 60-abdulazeez-fastapi.md

beebeckzzz opened the pull request:
Added timestamps

reshamas commented:
@beebeckzzz Thanks very much!
I updated the description above so it automatically closes the other open PR.

Confirming I have added the timestamps to the video.

beebeckzzz commented:
> @beebeckzzz Thanks very much! I updated the description above so it automatically closes the other open PR.
> 
> Confirming I have added the timestamps to the video.

@reshamas Happy to help. Keep up the great work! :)
